setup.cfg

```diff
@@ -68,8 +68,3 @@ omit=
     caluma/wsgi.py
     caluma/caluma_metadata.py
 show_missing = True
-
-[semantic_release]
```

Comment:
Shouldn't we keep the settings, so we can use semantic-release to at least help in the process?

Reply:
In a personal project, I use semantic-release without a config:

```shell
semantic-release changelog --noop --unreleased -D version_source=tag
```
With this, `semantic-release` will just lookup the last tag and go from there. The version variable needs to be changed manually.

Reply:
Fair enough. Nice work, let's merge it